app/assets/javascripts/discourse/app/lib/plugin-api.gjs

```diff
@@ -3,6 +3,7 @@
 // docs/CHANGELOG-JAVASCRIPT-PLUGIN-API.md whenever you change the version
 // using the format described at https://keepachangelog.com/en/1.0.0/.
 
+import { _addCategoryPropertyForUpdate } from "discourse/models/category";
 export const PLUGIN_API_VERSION = "2.1.1";
```

Comment:
This should bump the version number and add a line to `CHANGELOG-JAVASCRIPT-PLUGIN-API.md`. 

And the `import` statement should be moved below, no?

Reply:
I see, ok!